src/utils/sources-tree/utils.js

```diff
@@ -76,7 +76,7 @@ export function createParentMap(tree: Node): WeakMap<Node, Node> {
 }
 
 export function getRelativePath(path: string) {
-  const re = /(http(?:s?):\/\/(?:www\.)?[a-z0-9\-.]+)\/(.*)/i;
+  const re = /((http(?:s?)|webpack):\/\/((?:www\.)?[a-z0-9\-.]+)?)\/(.*)/i;
```

Comment:
i think we should generalize this to be any scheme: `file` `javascript` basically it can be any set of letters

Reply:
makes sense!